osu.Game.Rulesets.Osu/Skinning/Argon/ArgonMainCirclePiece.cs

```diff
@@ -62,7 +62,10 @@ public ArgonMainCirclePiece(bool withOuterFill)
             {
                 outerFill = new Circle // renders white outer border and dark fill
                 {
-                    Size = Size,
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    // Slightly inset to prevent bleeding outside the ring
+                    Size = Size - new Vector2(0.5f),
```

Comment:
I still see jaggies even with this. On my screen they only totally go away at `1f`.
| master | this PR | this PR but with `1f` |
| :-: | :-: | :-: |
| ![osu_2023-06-10_14-27-47](https://github.com/ppy/osu/assets/20418176/3322b351-dbbd-4d43-8da3-5ec425e03c2e) | ![osu_2023-06-10_14-29-03](https://github.com/ppy/osu/assets/20418176/9680c634-e66c-4832-b5b9-4a1092f5e2c7) | ![osu_2023-06-10_14-29-59](https://github.com/ppy/osu/assets/20418176/fceb8e0f-b673-408a-9a06-08764c5d0e51) |

Not sure if that's universal, or is it just resolution hell and we treat this as best-effort.

Difficulty I was testing on, for reference: https://osu.ppy.sh/beatmapsets/1866550#osu/3839769

Reply:
Makes sense. I originally had `1f`.